additional_tasks/task_15.py

```diff
@@ -0,0 +1,21 @@
+import random
+
+
+def get_odd_and_even(num_list, num_dict):
+    for num in num_list:
+        if num % 2 == 0:
```

Comment:
Ліпше писати таке з використанням `not`:
```suggestion
        if not num % 2:
```